coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -179,23 +179,25 @@ def _compile_multi_match_regex(strings):
     :param strings: The strings to match.
     :return:        A regex object.
     """
-    return re.compile('|'.join(re.escape(s) for s in strings))
+    return re.compile('^\s*(?P<marker>' +
+                      ('|'.join(re.escape(s) for s in strings)) +
+                      ')')
```

Comment:
Instead of compiling, we maybe just return the regex string? Then this function can still keep it's old pure functionality, and below at `begin_regex = ...` we compile it ourselves, adding the space-matching-stuff. What do you think?

Though like this it's also fine, I mean space matching is still understandable for this function :) Just an idea.